js/src/Collection/TodoCollection.js

```diff
@@ -0,0 +1,18 @@
+var Backbone = require('backbone');
+
+var TodoModel = require('../Model/TodoModel.js');
+
+var TodoCollection = Backbone.Collection.extend({
```

Comment:
Adding a comment to the whole class is also a good idea. Could be a simple comment, such as,
```
/**
 * TodoCollection is a collection of todos that belong to a given todo list.
 */
```